vllm/worker/worker.py

```diff
@@ -113,6 +117,34 @@ def __init__(
         # Initialize gpu_cache as embedding models don't initialize kv_caches
         self.gpu_cache: Optional[List[List[torch.Tensor]]] = None
 
+        # Torch profiler. Enabled and configured through env vars:
+        # VLLM_TORCH_PROFILER_DIR=/mnt/traces/
+        # view traces using https://ui.perfetto.dev/
```

Comment:
We should put the way of viewing traces to the profiling tutorial (not exists yet...)
```suggestion
        # Torch profiler. Enabled and configured through env vars:
        # VLLM_TORCH_PROFILER_DIR=/path/to/save/trace
```